builder/amazon/chroot/builder.go

```diff
@@ -121,7 +121,7 @@ type Config struct {
 	// Filters used to populate the source_ami field. Example:
 	//
 	//
-	//     ``` json
+	//     ```json
```

Comment:
Watch out; this is tricky and now always a fix: 
![Screenshot 2020-03-11 at 11 58 53](https://user-images.githubusercontent.com/1024852/76410123-b799bb00-638f-11ea-8796-abb340ef12fd.png)

https://deploy-preview-8868--packer-www.netlify.com/docs/builders/amazon-chroot.html#source_ami_filter

Reply:
I remember fighting with these a lot in #7724 and I visually checked on every doc change.

Reply:
Thanks for the back reference - I'll keep an eye and double check things. 
I saw this last night and thought to make this a WIP PR to give more time to review, but I fell a sleep :laughing:

Reply:
no worries 🙂